pkg/virt-launcher/virtwrap/converter/converter.go

```diff
@@ -1785,14 +1785,26 @@ func Convert_v1_VirtualMachineInstance_To_api_Domain(vmi *v1.VirtualMachineInsta
 				},
 			)
 		} else {
-			domain.Spec.Devices.Video = []api.Video{
-				{
-					Model: api.VideoModel{
-						Type:  "vga",
-						Heads: &heads,
-						VRam:  &vram,
+			// For AMD64 + EFI, use bochs. For BIOS, use VGA
+			if util.IsEFIVMI(vmi) {
+				domain.Spec.Devices.Video = []api.Video{
+					{
+						Model: api.VideoModel{
+							Type:  "bochs",
+							Heads: &heads,
+						},
```

Comment:
Would it make sense to add a unit test for that?

Also, I am thinking if it may cause troubles when you have current KubeVirt and some running VMs with EFI and the default vga display. Then KubeVirt is updated to a newer version, where for EFI the display defaults to bochs. Would it cause issues if I migrate such VMs?

Reply:
Yes, good point, I'll add a unit test.
Isn't this code ignored on migration? On migration, libvirt uses the source xml for the target domain, so it shouldn't be a problem. The VM will just switch to bochs next time it's rebooted.